INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-hu30kslfn7@apolo.us', username='regression-hu30kslfn7', password='A#S7^lOMBCmv') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-hu30kslfn7@apolo.us', username='regression-hu30kslfn7', password='A#S7^lOMBCmv')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-081xiozpw8@apolo.us', username='regression-081xiozpw8', password='UyGkCcYgM#v#') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-081xiozpw8@apolo.us', username='regression-081xiozpw8', password='UyGkCcYgM#v#')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-kai0grqs4p@apolo.us', username='regression-kai0grqs4p', password='YtJ&KpF!Y5&Z') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-kai0grqs4p@apolo.us', username='regression-kai0grqs4p', password='YtJ&KpF!Y5&Z')
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Verify Manager cannot change Admin role to Manager
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_27d0e6.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-hu30kslfn7@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_17562a.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_7ded24.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception ClientResponseError: 500, message='Internal Server Error', url='http://platform-admin:8080/apis/admin/v1/users/regression-hu30kslfn7?include=orgs&include=clusters&include=projects'. Path with query: /api/v1/config."}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-hu30kslfn7/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Timeout error for spinner
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_dd5542.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_fail_56964c.png
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_fail_74e18b.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Verify Manager cannot change Admin role to Manager
Exception: [AssertionError] during step: Login via UI
Message: [AssertionError] during step: Verify that Welcome new user page displayed
Message: Locator expected to be visible
Actual value: <element(s) not found> 
Call log:
  - Expect "to_be_visible" with timeout 5000ms
  - waiting for locator("h3.truncate.text-h3").filter(has_text="Welcome, regression-hu30kslfn7@apolo.us")

INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished
INFO     [🔧TEST CONFIG]:conftest.py:87 Loading test configuration
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:93 Creating data manager
INFO     [🔧TEST CONFIG]:conftest.py:123 Creating users manager
INFO     [🔧TEST CONFIG]:conftest.py:109 Creating API helper
INFO     [🔧TEST CONFIG]:conftest.py:117 Creating Apolo CLI instance
INFO     [🔧TEST CONFIG]:conftest.py:205 Continue using UserData(email='regression-hu30kslfn7@apolo.us', username='regression-hu30kslfn7', password='A#S7^lOMBCmv') as main test user...
INFO     [👤UsersManager]:users_manager.py:73 Setting main user to UserData(email='regression-hu30kslfn7@apolo.us', username='regression-hu30kslfn7', password='A#S7^lOMBCmv')
INFO     [🔧TEST CONFIG]:conftest.py:235 Continue using UserData(email='regression-081xiozpw8@apolo.us', username='regression-081xiozpw8', password='UyGkCcYgM#v#') as second test user...
INFO     [👤UsersManager]:users_manager.py:82 Setting second user to UserData(email='regression-081xiozpw8@apolo.us', username='regression-081xiozpw8', password='UyGkCcYgM#v#')
INFO     [🔧TEST CONFIG]:conftest.py:240 Continue using UserData(email='regression-kai0grqs4p@apolo.us', username='regression-kai0grqs4p', password='YtJ&KpF!Y5&Z') as third test user...
INFO     [👤UsersManager]:users_manager.py:91 Setting third user to UserData(email='regression-kai0grqs4p@apolo.us', username='regression-kai0grqs4p', password='YtJ&KpF!Y5&Z')
INFO     [📘TEST_INFO]:reporting.py:86 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:87 📝 TEST started: Verify Manager cannot change Admin role to Manager
INFO     [📘TEST_INFO]:reporting.py:88 ------------------------------------------------------------
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_1a5679.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-hu30kslfn7@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_b6c7fe.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_b2c23b.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [], "clusters": [], "projects": [], "admin_url": "https://api.dev.apolo.us/apis/admin/v1", "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-hu30kslfn7/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
[]
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-hu30kslfn7?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-hu30kslfn7", "email": "regression-hu30kslfn7@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-16T01:32:32.082979+00:00", "terms_accepted": true, "clusters": []}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-hu30kslfn7?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-hu30kslfn7',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-hu30kslfn7?include=orgs."}
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_992b3a.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_success_aa1a82.png
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_success_fe8f5a.png
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [🔧TEST CONFIG]:conftest.py:378 Navigating to: https://console.dev.apolo.us/
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": false, "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: <<User2>> :: Get second user
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: <<User2>> :: Get second user
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_get_second_user_success_3a6100.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: <<User2>> :: Login via UI
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: <<User2>> :: Click login button
INFO     [🎭 Playwright]:base_page.py:16 <AuthPage>: Click log in button
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: <<User2>> :: Click login button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_login_button_success_2640d7.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: <<User2>> :: Enter email
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter regression-081xiozpw8@apolo.us email
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: <<User2>> :: Enter email
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_email_success_416f9e.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: <<User2>> :: Enter password
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Enter ********* password
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: <<User2>> :: Enter password
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_enter_password_success_b6f0d6.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: <<User2>> :: Click Continue button
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Click continue button
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/config
Request Body:
<not available>
Response Body (first 1000 chars):
{"authorized": true, "orgs": [], "clusters": [], "projects": [], "admin_url": "https://api.dev.apolo.us/apis/admin/v1", "auth_url": "https://auth.dev.apolo.us/authorize", "token_url": "https://auth.dev.apolo.us/oauth/token", "logout_url": "https://auth.dev.apolo.us/v2/logout?returnTo=https://console.dev.apolo.us", "client_id": "q3I0OzzGnTDkhRmpeJ7WWgaTCucmVxTL", "audience": "https://api.dev.apolo.us", "callback_urls": ["http://127.0.0.1:54540", "http://127.0.0.1:54541", "http://127.0.0.1:54542"], "headless_callback_url": "https://api.dev.apolo.us/oauth/show-code", "success_redirect_url": "https://api.dev.apolo.us/oauth/login"}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/api/v1/users/regression-081xiozpw8/permissions?uri=blob:
Request Body:
<not available>
Response Body (first 1000 chars):
[]
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/invites/incoming
Request Body:
<not available>
Response Body (first 1000 chars):
[]
INFO     [🎭 Playwright]:base_page.py:16 <LoginPage>: Waiting for spinner to disappear
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 200] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-081xiozpw8?include=clusters&include=system_role
Request Body:
<not available>
Response Body (first 1000 chars):
{"name": "regression-081xiozpw8", "email": "regression-081xiozpw8@apolo.us", "first_name": null, "last_name": null, "created_at": "2025-09-16T01:37:42.693449+00:00", "terms_accepted": true, "clusters": []}
WARNING  [🔧TEST CONFIG]:conftest.py:420 [HTTP 500] GET https://api.dev.apolo.us/apis/admin/v1/users/regression-081xiozpw8?include=orgs
Request Body:
<not available>
Response Body (first 1000 chars):
{"error": "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-081xiozpw8',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/users/regression-081xiozpw8?include=orgs."}
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: <<User2>> :: Click Continue button
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_click_continue_button_success_9d497f.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: <<User2>> :: Verify that Welcome new user page displayed
INFO     [🎭 Playwright]:base_page.py:16 <WelcomeNewUserPage>: Check if page loaded
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: <<User2>> :: Verify that Welcome new user page displayed
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/verify_ui_page_displayed_success_b731c6.png
INFO     [📘TEST_INFO]:reporting.py:51 ✅ STEP completed: <<User2>> :: Login via UI
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (success): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_login_success_8e06c3.png
INFO     [📘TEST_INFO]:reporting.py:37 ▶️ STEP started: Add organization via API and reload page
INFO     [🔧DATA MANAGER]:test_data.py:56 Added organization: org_name=regression-org-fvtrilgq0ljl, gherkin_name=Default-organization
INFO     [🌐API_helper]:api_helper.py:86 POST https://api.dev.apolo.us/apis/admin/v1/orgs?skip_auto_add_to_clusters=false with data: {'name': 'regression-org-fvtrilgq0ljl'}
INFO     [🌐API_helper]:api_helper.py:190 Add org regression-org-fvtrilgq0ljl response: {'error': "Unexpected exception: (sqlalchemy.dialects.postgresql.asyncpg.ProgrammingError) <class 'asyncpg.exceptions.UndefinedColumnError'>: column users.email does not exist\n[SQL: SELECT users.name, users.metadata, users.email, users.version \nFROM users \nWHERE users.name = %s]\n[parameters: ('regression-org-fvtrilgq0ljl',)]\n(Background on this error at: https://sqlalche.me/e/14/f405). Path with query: /apis/admin/v1/orgs?skip_auto_add_to_clusters=false."}
ERROR    [📘TEST_INFO]:reporting.py:56 ❌ STEP failed: Add organization via API and reload page
INFO     [📘TEST_INFO]:reporting.py:225 📸 Screenshot captured and attached (fail): /home/runner/work/apolo-test-automation/apolo-test-automation/reports/screenshots/ui_add_org_api_fail_bebc6c.png
ERROR    [📘TEST_INFO]:reporting.py:95 ❌ TEST failed: Verify Manager cannot change Admin role to Manager
Exception: [AssertionError] during step: Add organization via API and reload page
Message: Expected HTTP 201 response but got 500!
INFO     [📘TEST_INFO]:reporting.py:98 ------------------------------------------------------------
INFO     [🌐API_helper]:api_helper.py:54 GET https://api.dev.apolo.us/apis/admin/v1/orgs
INFO     [🌐API_helper]:api_helper.py:165 Status: 200. Response: []
INFO     [🔧TEST CONFIG]:conftest.py:291 Cleaning up 0 organisations
INFO     [🔧TEST CONFIG]:conftest.py:321 Closing all browser sessions
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:327 Stopped Playwright
WARNING  [🔧TEST CONFIG]:conftest.py:337 Failed to close context: BrowserContext.close: Target page, context or browser has been closed
INFO     [🔧TEST CONFIG]:conftest.py:342 Closed browser (post-stop)
INFO     [🔧TEST CONFIG]:conftest.py:347 Browser cleanup finished